### DISCUSSION TITLE
Connect to internet via wired eth0, can not access LuCI with USB Wi-Fi

### DISCUSSION THREAD

**Discussion by tanuj-deqode** [SmoothWAN/SmoothWAN#13] [General & Help]
Hey, I am trying to connect the eth cable on the onboard eth port then I can not access the LuCI instead internet becomes available.

Trying to connect 2 https://www.amazon.in/gp/product/B008IFXQFU/ref=ppx_yo_dt_b_asin_title_o02_s00?ie=UTF8&psc=1 USB wifi, but those are not detected.

**Comment by TalalMash** [comment]
Hello, what platform are you using?
USB Wi-Fis are generally and intentionally not supported due to instability and performance issues with open source modules/drivers.
However, you can use the terminal and do `wifi detect`, you should see them in LuCI, if that did not work post `iw list` output here.

**Reply by TalalMash** [reply]
To add, thanks to the few donations, I'm getting few Ralink/Realtek cheap (~$2-5) USB Wi-Fi adapters to test and add auto configuration.
Yours seems to be RTL8188EUS? not sure if v1 or v2.

**Reply by tanuj-deqode** [reply]
I am not sure, I ordered three since it said it supports linux

**Reply by TalalMash** [reply]
Seems to be V2 and and a realtek.

**Reply by tanuj-deqode** [reply]
`Realtek RLT8188EUS` is the correct version.

**Comment by TalalMash** [comment]
Hello,
I had reliablity issues with the Realteks, I've added info for supported hardware: https://github.com/TalalMash/SmoothWAN/wiki/USB-WiFi-Adapters-in-Client-Mode

**Comment by loggerhead-turtle** [comment]
I stream for 4 hours straight and I worry about my cell phone being able to stay charged during live stream, as I don't know if it will pull enough power from the pi to stay charged. I want to setup a hotspot on my cell phone for 20 minutes during the stream so I can charge up the phone. How do I connect smoothwan to this Hotspot and then have speedify bond it? Thanks!

**Reply by TalalMash** [reply]
You can plug in a compatible Wi-Fi dongle to the Pi, more info in the Wiki : https://github.com/TalalMash/SmoothWAN/wiki/USB-WiFi-Adapters-in-Client-Mode

Excerpt:
## Supported models
- [AR9271](https://deviwiki.com/wiki/Atheros_AR9271#Table_of_Systems), tested models:
  - ALFA AWUS036NHA (beware of counterfeits)
 
You can use the search engine of any store for "AR9271" as they're widely popular.
## Setup
- Plug in the dongle to a USB port that will remian fixed, changing ports requires reconfiguration
- Restart/Reboot SmoothWAN
- Configure in Network -> Wireless 
- Hit `Scan` and follow through, tick `Replace existing configuration` and set firewall to RED zone
- Done! Speedify will automatically detect and add the new interface.
